index.d.ts

```diff
@@ -0,0 +1,89 @@
+declare module "moleculer-apollo-server" {
+	import { ServiceSchema, Context } from "moleculer";
+	import { Config } from "apollo-server-core";
+	import { OptionsUrlencoded } from "body-parser";
+	import { SchemaDirectiveVisitor, IResolvers } from "graphql-tools";
+
+	export {
+		GraphQLUpload,
+		GraphQLExtension,
+		gql,
+		ApolloError,
+		toApolloError,
+		SyntaxError,
+		ValidationError,
+		AuthenticationError,
+		ForbiddenError,
+		UserInputError,
+		defaultPlaygroundOptions,
+	} from "apollo-server-core";
+
+	export * from "graphql-tools";
+
+	export interface ApolloServerOptions {
+		path: string;
+		disableHealthCheck: boolean;
+		onHealthCheck: () => {};
+	}
+
+	export class ApolloServer {
+		createGraphQLServerOptions(req: any, res: any): Promise<any>;
+		createHandler(options: ApolloServerOptions): void;
+		supportsUploads(): boolean;
+		supportsSubscriptions(): boolean;
+	}
+
+	export interface ActionResolverSchema {
```

Comment:
I think one of my previous comments got missed.
This should also allow properties:
```
nullIfError?: boolean;
params?: { [key: string]: any };
```